src/sqliteformat.cpp

```diff
@@ -1298,10 +1340,10 @@ bool SqliteFormat::Private::insertCalendarProperty(const QByteArray &id,
 }
 //@endcond
 
-Notebook::Ptr SqliteFormat::selectCalendars(sqlite3_stmt *stmt, bool *isDefault)
+Notebook* SqliteFormat::selectCalendars(sqlite3_stmt *stmt, bool *isDefault)
```

Comment:
Hm, does the pointer return value save much when the result gets deleted anyway? Perhaps could alternatively just return a Notebook instance which wouldn't need the deletion.

Reply:
I agree, it's now returning a Notebook instance. instead of a pointer on a Notebook instance.